Geometry_Engine/Create/ShapeProfile.cs

```diff
@@ -194,5 +481,19 @@ private static List<ICurve> MirrorAboutLocalZ(this List<ICurve> curves)
         }
 
         /***************************************************/
+
+        private static void InvalidRatioError(string inputs)
+        {
+            Engine.Reflection.Compute.RecordError("The ratio between " + inputs + " makes section inconceivable");
+        }
+
+        /***************************************************/
+
+        private static void InvalidRatioError2(string first, string second)
```

Comment:
This is kind of on me as well, but the 2 is not needed here. Method overloading will take care of it with different number of inputs.